masschallenge/opsworks-web-python
# [AC-9698] Timebox: Prevent/monitor front-end deploy failures

## [AC-9698](https://masschallenge.atlassian.net/browse/AC-9698)

**Changes introduced**
- add  buildout logs to stack deploy logs

**Testing**
- Use this branch as the custom Chef cookbooks revision
- Confirm that you can see buildout logs on the deploy logs


[Sibling PR](https://github.com/masschallenge/accelerate/pull/3398)
